src/main/java/com/sap/oss/phosphor/fosstars/tool/format/OssRulesOfPlayRatingMarkdownFormatter.java

```diff
@@ -377,4 +423,24 @@ public static String format(Label label) {
 
     return map;
   }
+  
+  /**
+   * Load rule documentation URI from a file.
+   *
+   * @param path A path to the configuration file.
+   * @return The rule documentation URL.
+   * @throws IOException If something went wrong.
+   */
+  private static String loadRuleDocumentationUrlFrom(Path path) throws IOException {
+    JsonNode config = Yaml.mapper().readTree(Files.newBufferedReader(path));
+    if (!config.has("documentationUrl")) {
+      return StringUtils.EMPTY;
+    }
+
+    if (!config.isObject()) {
+      throw new IOException("Oops! Configuration is not an object!");
+    }
+
+    return config.get("documentationUrl").asText();
```

Comment:
It would be better to close the reader. I'll fix it later.

```suggestion
    try (BufferedReader reader = Files.newBufferedReader(path)) {
      JsonNode config = Yaml.mapper().readTree(reader);
      if (!config.has("documentationUrl")) {
        return StringUtils.EMPTY;
      }

      if (!config.isObject()) {
        throw new IOException("Oops! Configuration is not an object!");
      }

      return config.get("documentationUrl").asText();
    }
```